### ISSUE TITLE
Render the imprint from a markdown file.

### ISSUE THREAD

**Issue by kiliw** [hardforkio/homepage#115]
Similar to #112 we could render the imprint page from a markdown file.

Use netlify.cms

**Comment by Bakkhos** [comment]
I ended up doing this:
- [configuring netlify CMS](https://github.com/hardforkio/homepage-new/blob/ef7806c87b177d5f1cd86264e0500113a53c1bf7/static/admin/config.yml#L10-L38) to store the [imprint data](https://github.com/hardforkio/homepage-new/blob/ef7806c87b177d5f1cd86264e0500113a53c1bf7/src/data/imprint/imprint.json#L1-L20) as JSON (not markdown). However, the content field is markdown
- [importing this json into the source code as a typed object](https://github.com/hardforkio/homepage-new/blob/ef7806c87b177d5f1cd86264e0500113a53c1bf7/src/data/imprint/index.tsx#L5-L12)
- and rendering the imprint with a[ react component ](https://github.com/hardforkio/homepage-new/blob/ef7806c87b177d5f1cd86264e0500113a53c1bf7/src/components/Imprint/index.tsx#L7-L20)that takes markdown as a [prop](https://github.com/hardforkio/homepage-new/blob/ef7806c87b177d5f1cd86264e0500113a53c1bf7/src/components/Imprint/index.tsx#L8-L9)

So basically, we do render the imprint from markdown, but not from a markdown file. That said, the imprint's data could be nicely represented as a markdown file. Maybe it would have been the better approach to configure the CMS to generate an imprint.md (with the imprint [metadata](https://github.com/hardforkio/homepage-new/blob/ef7806c87b177d5f1cd86264e0500113a53c1bf7/src/data/imprint/index.tsx#L5-L10) in the frontmatter and the content in the body) and to set up gatsby to generate the imprint from this file.
